integreat_cms/cms/templates/pages/page_form.html

```diff
@@ -175,7 +175,7 @@ <h1 class="heading">
             </div>
             {% if page %}
                 <div id="right-sidebar-column"
-                     class="3xl:col-end-3 4xl:col-end-auto flex flex-wrap flex-col gap-2">
+                     class="mt-4 lg:mt-0 3xl:col-end-3 4xl:col-end-auto flex flex-wrap flex-col gap-2">
```

Comment:
```suggestion
                     class="3xl:col-end-3 4xl:col-end-auto flex flex-wrap flex-col gap-2">
```

Reply:
These classes were added when I rebased the branch, but I can remove them again :)